tests/kani/ThreadLocalRef/main.rs

```diff
@@ -9,9 +9,8 @@ thread_local! {
 
 #[kani::proof]
```

Comment:
Maybe also add a test where you do a series of operation on a thread local? And maybe one where the type is a struct not just a primitive?

Reply:
Done. I added a test with `&'static str` which is a fat pointer, so compiles to a struct.